.travis.yml

```diff
@@ -1,21 +1,23 @@
 language: python
 
+sudo: false
+
 python:
   - "2.7"
   - "2.6"
+  - "3.4"
+
+addons:
+  apt:
+    packages:
+      - libgeos-c1
+      - libxml2-dev
 
 install:
-  - sudo apt-get install -y libgeos-c1
   - pip install . --use-mirrors
-  - pip install -r ./requirements-standalone.txt
-  - pip install -r ./requirements-dev.txt
-  - python ./setup.py -q install
-
-before_script:
-  - sudo add-apt-repository ppa:gcpp-kalxas/ppa-tzotsos -y
```

Comment:
It looks like moving to Travis' [container-based infrastructure](http://docs.travis-ci.com/user/workers/container-based-infrastructure/) results in the libxml2 upgrade, i.e.:

- master: https://travis-ci.org/geopython/pycsw/jobs/78919311
- this PR: https://travis-ci.org/geopython/pycsw/jobs/80046171#L83

Having said this I'm +1 with pushing the `.travis.yml` changes immediately in advance of this PR (or even as an isolated PR), given it removes the requirement to manage our own libxml2 for the tests to correctly pass.

Thoughts?